content/docs/user-guide/experiment-management/sharing-experiments.md

```diff
@@ -1,121 +1,83 @@
 # Sharing Experiments
 
-There are two types of remotes that can store experiments. Git remotes are
-distributed copies of the Git repository, for example on GitHub or GitLab.
+If your team uses Git server or hosting (e.g. GitHub, GitLab, etc.) to
+collaborate on projects, you can also use them to save and share DVC
+Experiments.
 
-[DVC remotes](/doc/command-reference/remote) on the other hand are
-storage-specific locations (e.g. Amazon S3 or Google Drive) which we can
-configure with `dvc remote`. DVC uses them to store and fetch large files that
-don't normally fit inside Git repos.
+> You will need both [Git remotes] and DVC
```

Comment:
I still don't see a strong reason for these notes to be honest ... we mention GH already before ... what are the chances people won't have Git remote?

Reply:
Yeah, I removed this but just incorporated the link to Git remotes into the details section below.